pkg/providers/amifamily/ami.go

```diff
@@ -119,7 +119,9 @@ func (p *DefaultProvider) List(ctx context.Context, nodeClass *v1beta1.EC2NodeCl
 	}
 	amis.Sort()
 	if p.cm.HasChanged(fmt.Sprintf("amis/%s", nodeClass.Name), amis) {
-		logging.FromContext(ctx).With("ids", amis, "count", len(amis)).Debugf("discovered amis")
+		logging.FromContext(ctx).With(
+			"ids", pretty.Slice(lo.Uniq(lo.Map(nodeClass.Status.AMIs, func(a v1beta1.AMI, _ int) string { return a.ID })), 25),
```

Comment:
Do we ever expect there to be more than 25 amis?

Reply:
It is possible for a customer to specify more then 25 AMIs with using tags

Reply:
No, because these AMIs have to be unique by requirements

Reply:
Removed truncating the AMI IDs